boto/dynamodb2/table.py

```diff
@@ -439,7 +446,134 @@ def update(self, throughput, global_indexes=None):
         self.connection.update_table(
             self.table_name,
             provisioned_throughput=data,
-            global_secondary_index_updates=gsi_data
+            global_secondary_index_updates=gsi_data,
+        )
+        return True
+
+    def create_global_secondary_index(self, global_index):
+        """
+        Creates a global index in DynamoDB after the table has been created.
+
+        Requires a ``global_indexes`` parameter, which should be a
+        ``GlobalBaseIndexField`` subclass representing the desired index.
+
+        Returns ``True`` on success.
+
+        Example::
+
+            # To create a global index
+            >>> users.create_global_secondary_index(
+            ...     global_index=GlobalAllIndex(
+            ...         'TheIndexNameHere', parts=[
+            ...             HashKey('requiredHashkey', data_type=STRING),
+            ...             RangeKey('optionalRangeKey', data_type=STRING)
+            ...         ],
+            ...         throughput={
+            ...             'read': 2,
+            ...             'write': 1,
+            ...         })
+            ... )
+            True
+
+        """
+
+        gsi_data = None
+        gsi_data_attr_def = None
+
+        if global_index:
+            gsi_data = []
+            gsi_data_attr_def = []
+
+            gsi_data.append({
+                "Create": global_index.schema()
+            })
+
+            for attr_def in global_index.parts:
+                gsi_data_attr_def.append(attr_def.definition())
+
+        self.connection.update_table(
+            self.table_name,
+            global_secondary_index_updates=gsi_data,
+            attribute_definitions=gsi_data_attr_def
+        )
+        return True
+
+    def delete_global_secondary_index(self, global_index_name):
+        """
+        Deletes a global index in DynamoDB after the table has been created.
+
+        Requires a ``global_index_name`` parameter, which should be a simple
+        string of the name of the global secondary index.
+
+        Returns ``True`` on success.
+
+        Example::
+
+            # To delete a global index
+            >>> users.delete_global_secondary_index('TheIndexNameHere')
+            True
+
+        """
+
+        gsi_data = None
+
+        if global_index_name:
```

Comment:
Same here: `global_index_name` is a requirement for this method to fully work. So if empty: `return False` and add a line in the logs ("You need to provide the index name to delete_global_secondary_index", or something along those lines.)